velox/exec/HashProbe.h

```diff
@@ -229,28 +232,28 @@ class HashProbe : public Operator {
   /// matches.
   SemiJoinTracker semiJoinTracker_;
 
-  // Keeps track of returned results between successive batches of
-  // output for a batch of input.
+  /// Keeps track of returned results between successive batches of
+  /// output for a batch of input.
   BaseHashTable::JoinResultIterator results_;
 
   RowVectorPtr output_;
 
-  // Input rows with no nulls in the join keys.
+  /// Input rows with no nulls in the join keys.
   SelectivityVector nonNullRows_;
 
-  // Input rows with a hash match. This is a subset of rows with no nulls in the
-  // join keys and a superset of rows that have a match on the build side.
+  /// Input rows with a hash match. This is a subset of rows with no nulls in
+  /// the join keys and a superset of rows that have a match on the build side.
   SelectivityVector activeRows_;
 
   bool finished_{false};
 
-  // True if passingInputRows is up to date.
+  /// True if passingInputRows is up to date.
   bool passingInputRowsInitialized_;
 
-  // Set of input rows for which there is at least one join hit. All
-  // set if right side optional. Used when loading lazy vectors for
-  // cases where there is more than one batch of output or join filter
-  // input.
+  /// Set of input rows for which there is at least one join hit. All
```

Comment:
Thank you for updating the comment style. This diff is a bit tricky and would benefit from a careful review. Would you consider extracting refactoring into a separate PR to make reviewing easier?

Reply:
Will do.